Source/Core/VideoBackends/D3D12/D3DBase.cpp

```diff
@@ -332,7 +332,7 @@ std::vector<DXGI_SAMPLE_DESC> EnumAAModes(IDXGIAdapter* adapter)
 
 D3D_FEATURE_LEVEL GetFeatureLevel(IDXGIAdapter* adapter)
 {
-	return D3D_FEATURE_LEVEL_11_0;
```

Comment:
Shouldn't this function actually check the feature level instead of returning a constant?